### PR TITLE
documentation and code maintenance

### PR DESCRIPTION
**Pull request by HolQue**
* Fixed package name in CConfig (QConnectBase now; previously robotframework-qconnect-base). The package name defined in CConfig must fit to the folder name containing the sources! Otherwise the setup will not run properly.
* added 'config' to package exclude filter
* Sphinx import path fix
* code maintenance (company name)
* maintained documentation

### CODE REVIEW

**Comment by test-fullautomation** [comment]
Hi Holger,
as discussed I don't merge the pull request now.
The package name must be robotframework-qconnect-base, the import on robot level must be QConnectBase.
Please provide a commit for this.
Thank you,
 Thomas

File: config/CConfig.py [hunk 1]
```diff
@@ -85,7 +90,7 @@ def __init__(self, sReferencePath="."):
         self.__dictConfig['sReferencePath'] = self.__sReferencePath # only to have the possibility to print out all values only with help of 'self.__dictConfig'
 
         # 1. basic setup stuff
-        self.__dictConfig['sPackageName']                = "robotframework-qconnect-base"
+        self.__dictConfig['sPackageName']                = "QConnectBase"
```
Comment (test-fullautomation) [inline config/CConfig.py]: Please refer to my comment https://github.com/test-fullautomation/robotframework-qconnect-base/pull/6#issuecomment-1062809012

Comment (test-fullautomation) [reply config/CConfig.py]: Hi Holger,
I have to apologize or the name-confusion. I corrected the comment.
The import mus be QConnectBase as it is now. I accidentaly wrote it wrong before.
Sorry for this additional work I produced.
Thank you,
 Thomas

Comment (HolQue) [reply config/CConfig.py]: The latest commit will fix this.

**Review by test-fullautomation — APPROVED** [review]
Hi Holger,
very good,
Thank you,
 Thomas

File: config/CConfig.py (hunk 1 above)
Comment (test-fullautomation) [reply config/CConfig.py]: Hi Holger,
thank you, I merge now.
Thomas

**Pull request merged**